recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -11,7 +11,17 @@ CMDLINE_DWC_OTG ?= "dwc_otg.lpm_enable=0"
 CMDLINE_ROOT_FSTYPE ?= "rootfstype=ext4"
 CMDLINE_ROOTFS ?= "root=/dev/mmcblk0p2 ${CMDLINE_ROOT_FSTYPE} rootwait"
 
-CMDLINE_SERIAL ?= "${@oe.utils.conditional("ENABLE_UART", "1", "console=serial0,115200", "", d)}"
+# revert the original serial setup
+def serial_comms(d):
```

Comment:
I'm not sure how this is relevant for this commit. Maybe I'm missing something but this is already flexible by using the `CMDLINE_SERIAL` variable. What is your use case here?

Reply:
More experimentation shows that I do not actually need the UART enabled at all for a serial connection. This change will be backed out.

Originally I was under the impression I needed the string " console=tty1" appended to the CMDLIE_SERIAL, and I thought this was the cleanest way to do that. I am open to suggestions if there is a better approach to that (for the next time this might come up)

Reply:
@ajpenner can you please clean up and include the documentation here too? Closing https://github.com/agherzan/meta-raspberrypi/pull/1001 assuming that you pull include it as part of this PR.

Reply:
I have reverted the CMDLINE_SERIAL change since it was not needed. I have added the documentation here as well.